pkgs/applications/office/libreoffice/default.nix

```diff
@@ -57,8 +60,9 @@ in (stdenv.mkDerivation rec {
 
   # For some reason librdf_redland sometimes refers to rasqal.h instead
   # of rasqal/rasqal.h
-  NIX_CFLAGS_COMPILE = "-I${librdf_rasqal}/include/rasqal"
-    + stdenv.lib.optionalString stdenv.isx86_64 " -mno-fma";
+  NIX_CFLAGS_COMPILE = [
```

Comment:
Environment variables need to be strings, this will be an error with structured attrs.

Reply:
I had a read through https://nixos.mayflower.consulting/blog/2020/01/20/structured-attrs/ which was really quite interesting. Don't we consider a list a primitive type on nix?

Reply:
Reasoning here: https://github.com/NixOS/nixpkgs/pull/72074#issuecomment-609435309

Reply:
Thanks for the pointers, I'll open a separate issue to address this.